src/parse.py

```diff
@@ -45,27 +45,43 @@ def parse_usbank(record):
         "date": record["Date"],
         "description": record["Name"],
         "amount": record["Amount"],
-        # no category :(
+        "category": None,  # No category
         "source": "usbank",
     }
 
 
+PARSER_DICT = {
+    "amex": parse_amex,
+    "chase": parse_chase,
+    "capital_one": parse_capital_one,
+    "usbank": parse_usbank,
+}
+
+
 def default_parser(record):
     return record
 
 
-def get_parser(filename):
+def get_card(filename):
```

Comment:
I was also thinking about if it's possible to make this function better, I wasn't a big fan of the 4 if statements we had. I like the idea of a parser dict but we still have the list of if statements. What do we gain by doing in 2 steps what we did before in 1?

Reply:
This again goes back to the idea of (not) determining file format from filename prefix. By splitting it into two, we avoid that.